### PR TITLE
Adding a folder for Javad

### PR DESCRIPTION
**Pull request by jaavad**

### CODE REVIEW

File: content/authors/Javad Moein/avatar.jpg
Comment (pointW) [inline content/authors/Javad Moein/avatar.jpg:1]: Can you put your face at the center of your avatar? Please check the preview https://deploy-preview-101--helpinghandslab.netlify.app/

File: content/authors/Javad Moein/_index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+# Username (this should match the folder name)
+authors:
+- Javad Moein
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position
+role: Graduate Student, CS, AI
```
Comment (pointW) [inline content/authors/Javad Moein/_index.md:13]: Can you make your program and major clear? I.e., M.S. student, CS or M.S. student, AI

**Review by pointW — CHANGES_REQUESTED** [review]

File: content/authors/Javad Moein/avatar.jpg
Comment (jaavad) [reply content/authors/Javad Moein/avatar.jpg:1]: Sure, I did it- [adding edited Photo]

File: content/authors/Javad Moein/_index.md (hunk 1 above)
Comment (jaavad) [reply content/authors/Javad Moein/_index.md]: Done

**Review by pointW — APPROVED** [review]

**Pull request merged**